src/osd/pg_scrubber.h

```diff
@@ -560,6 +568,16 @@ class PgScrubber : public ScrubPgIF, public ScrubMachineListener {
    *  discarded.
    */
   epoch_t m_epoch_start{0};  ///< the actual epoch when scrubbing started
+
+  /**
+   *  (replica) a tag identifying a specific scrub "session". Incremented whenever the
+   *  Primary releases the replica scrub resources.
+   *  When the scrub session is terminated (even if the interval remains unchanged, as
+   *  might happen following an asok no-scrub command), stale scrub-resched messages
+   *  triggered by the backend will be discarded.
+   */
+  std::atomic<Scrub::act_token_t> m_current_token{1};
```

Comment:
I don't see any users of m_current_token that aren't under the pg lock.  Does this need to be atomic?

Reply:
Hmmm. I think you are right.
Fixing.